src/main/resources/templates/item.html

```diff
@@ -8,13 +8,14 @@
     <div th:replace="main.html::phase"></div>
     <div class="grid-row">
         <div class="column-two-thirds">
+            <p></p>
```

Comment:
added this empty `<p>` tag temporarily to right align the attribution so that it is consistent with other pages.